src/add-remove.test.js

```diff
@@ -0,0 +1,20 @@
+const myModules = require('./add-remove.js');
```

Comment:
- [ ] [optional] it's best to use the ES6 syntax like ```import```/```export``` instead of ```require```, for best practices and consistency over all your code